homework-unit7/chat/main.css

```
@@ -0,0 +1,252 @@
+*{
```

```suggestion
* {
```

Trước dấu `{` hãy thêm 1 dấu cách